index.html

```diff
@@ -1 +1,38 @@
-My Page
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>Facebook Clone</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" type="text/css" media="screen" href="homepage.css" />
+  </head>
+  <body>
+    <div class="container">
```

Comment:
You did a good job on this, you have too much-nested div tags in you page navigation and you did't make use of proper `semantic tags`